outline/intro.md

```diff
@@ -195,7 +195,7 @@ Introduction to Programming with Clojure
 
 **R**ead, **E**val, **P**rint, **L**oop
 
-![Instarepl](img/instarepl.png)
+![Nightcode's repl](img/repl.png)
```

Comment:
The screenshot currently depicts the following:

```clojure
user=> (str "Hello World!")
"Hello World!"
user=> (+ 3 4)
7
user=> ;; this line is ignored
user=>
```

I think the first interaction should be the simplest possible interaction: providing a value and having it echoed. I suggest changing the first expression from `(str "Hello World!")` to `"Hello World!"`.